handbook/product/onboarding/index.md

```diff
@@ -17,7 +17,7 @@ Welcome to Sourcegraph! As a member of the product team, it is your job to be th
 - Schedule a recurring [1-1](../../leadership/1-1.md).
 - Schedule daily check-ins for the first week at Sourcegraph to keep up with onboarding and to create space for answering any questions that might come up.
 - Create a 1-1 doc and add initial discussion items
-   - Onboarding doc - YAYYYYYYY WELCOME!!!!! 🎉
+   - Onboarding doc - YAYYYYYYYYYY WELCOME!!!!! 🎉
```

Comment:
VIP change.

Reply:
Gotta have a little fun while revamping the bizops handbook pages!